.eslintrc

```diff
@@ -1,37 +1,38 @@
 {
-	"parser": "babel-eslint",
-	"env": {
-		"browser": true,
-		"node": true
-	},
-	"plugins": [
-		"babel",
-		"nodeca",
-		"react"
-	],
-	"rules": {
-		"strict": 0,
-		"camelcase": 0,
-		"curly": 0,
-		"indent": [0, "tab"],
-		"nodeca/indent": [2, "tabs", 1],
-		"eol-last": 1,
-		"no-shadow": 0,
-		"no-redeclare": 1,
-		"no-extra-bind": 1,
-		"no-empty": 0,
-		"no-process-exit": 1,
-		"no-underscore-dangle": 0,
-		"no-use-before-define": 0,
-		"no-unused-vars": 0,
-		"consistent-return": 0,
-		"no-inner-declarations": 1,
-		"no-loop-func": 1,
-		"quotes": [2, "double"],
-		"space-before-function-paren": [2, "never"],
-		"jsx-quotes": 1,
-		"react/jsx-no-undef": 1,
-		"react/jsx-uses-react": 1,
-		"react/jsx-uses-vars": 1
-	}
+  "parser": "babel-eslint",
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "extends": "airbnb",
+  "plugins": [
+    "babel",
+    "nodeca",
+    "react"
+  ],
+  "rules": {
```

Comment:
Do we really need custom rules here? Everything should be included from airbnb I think.